docs/resources/aws_flow_log.md

```diff
@@ -0,0 +1,160 @@
+---
+title: About the aws_flow_log Resource
+---
+
+# aws_flow_log
+
+Use the `aws_flow_log` InSpec audit resource to test detailed properties of an individual Flow Log.
+
+<br>
+
+## Syntax
+
+An `aws_flow_log` resource block uses resource parameters to search for a Flow Log, and then tests that Flow Log.  If no Flow Logs match, no error is raised, but the `exists` matcher will return `false` and all properties will be `nil`.
+
+    # Ensure the flow log is active
+    describe aws_flow_log(flow_log_id: 'fl-12345678') do
+      its('flow_log_status') { should eq 'ACTIVE' }
+    end
+
+    describe aws_flow_log(vpc_id: 'fl-12345678') do
+      it { should exist }
+    end
+
+    describe aws_flow_log(subnet_id: 'fl-12345678') do
+      it { should exist }
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+As this is the initial release of `aws_flow_log`, its limited functionality precludes examples.
```

Comment:
I disagree - since you'd added most of the properties, you can certainly include some examples.